### PR TITLE
update core to 5.23.5 and sync to 4.7.8

### PR DESCRIPTION
**Pull request by bmunkholm**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,20 +1,22 @@
-x.y.z Release notes (yyyy-MM-dd)
+3.18.1 Release notes (yyyy-MM-dd)
```
Comment (tgoyne) [inline CHANGELOG.md]: This is set as part of making a release, and changing it before that breaks the script that automatically sets it. 3.18.1 is also not the correct version number for the next release.

File: Package.swift [hunk 2]
```diff
@@ -3,8 +3,8 @@
 import PackageDescription
 import Foundation
 
-let coreVersionStr = "5.23.3"
-let cocoaVersionStr = "3.18.0"
+let coreVersionStr = "5.23.5"
+let cocoaVersionStr = "3.18.1"
```
Comment (tgoyne) [inline Package.swift]: This is set by the release script.

File: Realm/Realm-Info.plist [hunk 3]
```diff
@@ -17,11 +17,11 @@
 	<key>CFBundlePackageType</key>
 	<string>FMWK</string>
 	<key>CFBundleShortVersionString</key>
-	<string>3.18.0</string>
+	<string>3.18.1</string>
```
Comment (tgoyne) [inline Realm/Realm-Info.plist]: This is set by the release script.

File: dependencies.list [hunk 4]
```diff
@@ -1,4 +1,4 @@
-VERSION=3.18.0
-REALM_CORE_VERSION=5.23.4
-REALM_SYNC_VERSION=4.7.5-catalyst.3
+VERSION=3.18.1
```
Comment (tgoyne) [inline dependencies.list]: This is set by the release script.

File: dependencies.list [hunk 5]
```diff
@@ -1,4 +1,4 @@
...
+REALM_CORE_VERSION=5.23.5
+REALM_SYNC_VERSION=4.7.8-catalyst.3
```
Comment (tgoyne) [inline dependencies.list]: This is not a release that exists.

Comment (bmunkholm) [reply dependencies.list]: Are the catalyst releases done manually for now?

**Comment by bmunkholm** [comment]
HaHa - yeah not too surprised I did a lot wrong here :-) Feel free to just pick the core version and changelog entries then in another commit if that's easiest.

File: dependencies.list (hunk 5 above)
Comment (tgoyne) [reply dependencies.list]: 4.7.5-catalyst was 4.7.5 rebuilt with the build system changes needed to support catalyst. Those have now been merged and the normal 4.7.8 release supports it.

**Pull request closed without merging**